docsk8s/quick-start/ls-k8s-quick-start.asciidoc

```diff
@@ -184,16 +273,105 @@ After finishing with this demo, you can run the following command to remove all
  
 [source,sh]
 --
-kubectl delete service,pods,deployment,hpa,configmap,secret,beat,elasticsearch,kibana -l app=logstash-demo
+kubectl delete service,pods,deployment,configmap,secret,beat,elasticsearch,kibana -l app=logstash-demo
 --
 
+
+[float]
+[[qs-next-steps]]
+=== Next Steps
+
+[float]
+[[qs-external-elasticsearch]]
+==== Send logs to an External Elasticsearch
+
+
+Rather than sending to an elasticsearch cluster in the same Kubernetes cluster as logstash, you may want to send data to an Elastic cloud instance.
+
+
+[float]
+[[qs-send-to-elastic-cloud]]
+===== Sending to Elastic Cloud
+
+To connect to Elastic cloud, we will only need the logstash based components - no need to include the `elasticsearch` or `kibana` components from the walkthrough
+
+We will need to amend the `Deployment`/`StatefulSet` to set `CLOUD_ID` and `API_KEY` environment variables with the appropriate value for your cloud instance.
+
+One way to do this is to create a link:https://kubernetes.io/docs/concepts/configuration/secret/[secret] to store `CLOUD_ID` and `API_KEY`:
+
+
+[source,yaml]
+--
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ess_secret
+type: Opaque
+data:
+  cloud_id: PENMT1VEX0lEPg== <1>
+  password: PEFQSV9LRVk+
+--
+<1> base64 representation of `cloud_id` and `api_key` for your elastic cloud instance - created by
++
+[source,sh]
+--
+echo -n '<CLOUD_ID>' | base64
+echo -n '<API_KEY>' | base64
+--
+
+
+Mount the secrets in the `Deployment`/`StatefulSet`
+
+
+[source,yaml]
+--
+env:
+  - name: CLOUD_ID
+      valueFrom:
+        secretKeyRef:
+          name: ess_secret
+          key: cloud_id
+    - name: API_KEY
+      valueFrom:
+        secretKeyRef:
+          name: ess_secret
+          key: api_key
+
+--
+
+And we will amend the pipeline definition `ConfigMap` to change the destination of the elasticsearch output to the cloud instance.
+
+[source,yaml]
+--
+    output {
+      elasticsearch {
+        cloud_id => "CLOUD_ID"
+        api_key => "API_KEY"
+        ssl => true
+      }
+--
+
+[float]
+[[qs-scale-logstash]]
+==== Scale Logstash with Horizontal Pod Autoscaler
+
+For a simple Logstash setup, without <<ls-k8s-persistent-storage, persistent storage>>, or <<ls-k8s-design-for-plugins, plugins that require the storing of local state>>, we can introduce a simple <<qs-autoscaler, horizontal pod autoscaler>>.
+
+Apply the autoscaler as follows:
+
+[source,bash]
+--
+kubectl apply -f "004-hpa.yaml"
+--
+
+NOTE: When using more than one logstash pod, use the link:https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-autodiscover.html#_kubernetes[autodiscover] features of beats to monitor the different logstash pods, otherwise only one logstash pod will be monitored.
```

Comment:
Do you think it makes sense to link to https://github.com/elastic/logstash/blob/05bfaff799d3cb52b2d0af51d9a9a51f15a0501f/docsk8s/administering/ls-k8s-stack-monitoring.asciidoc#L17 where Kaise mentioned autodiscover in the context of stack monitoring?

Reply:
Sure

Reply:
```suggestion
NOTE: If you are using more than one {ls} pod, use the https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-autodiscover.html#_kubernetes[beats autodiscover] features to monitor them. Otherwise, only one {ls} pod is monitored.
See the <<stack monitoring with ECK,monitor-with-ECK>> docs for details on how to use autodiscover with {metricbeat} and {ls}.

```